tests/ConnectionTest.php

```diff
@@ -40,7 +40,7 @@ function ($errno, $errstr, $errfile, $errline, $errcontext) use ($testCase, &$ha
                 $handlerInvoked = true;
 
                 $testCase->assertSame(E_USER_WARNING, $errno);
-                $testCase->assertSame(
+                $testCase->assertContains(
```

Comment:
Note: I am deliberately doing a substring match here, because the code and the message might differ per platform or the standard library in use.